### ISSUE TITLE
configure-git doesn't work on Windows10

### ISSUE THREAD

**Issue by mwasilew** [foundriesio/fioctl#263]
When configuring git helper on windows there is a mismatch between expected and actual git command. Following the docs step results in the following setup:

execute as administrator
```
C:\Windows\system32\fioctl-windows-amd64.exe configure-git
Symlinking C:\Users\Milosz\fio\fio-windows-amd64.exe to C:\Program Files\Git\cmd\git-credential-fio.exe
```

At the same time git's global config is configured as follows:
```
credential.https://source.foundries.io.helper=fio.exe
```

When using this setup, git credential manager displays the following dialog:
![fioctl_dialog](https://github.com/foundriesio/fioctl/assets/618901/84a06381-675a-442d-9e9c-eec39dde7e90)

This issue can be fixed by reconfiguring git. This can be done by:
```
git config --global --edit
```
and changing fio.exe to git-credential-fio.exe

For some reason running 
```
git config --global credential.https://source.foundries.io.helper=git-credential-fio.exe
```
doesn' work.

I believe the root of the problem is here:
https://github.com/foundriesio/fioctl/blob/main/subcommands/git/cmd.go#L77

Possibly change this line to:
```
helperName = dst
```

**Comment by doanac** [comment]
weird. I've tested this in the past. However, I think there are lots of options to select from while installing Git. I think one of those options might be messing this up.

**Comment by mwasilew** [comment]
It's very weird. In this scenario I installed both fioctl and git on a fresh system - it didn't work. When I made some changes to make it work I wasn't able to go back to the "broken" state. Git kept the old settings in some cache which was not available to the user to even see.

**Comment by StealthyCoder** [comment]
I made a quick PR #264 . I did the same as you @mwasilew , new clean system with fresh [Git](https://git-scm.com/download/win) not [Git for Windows](https://gitforwindows.org/). I chose some non default options like no symlinks and so on. Then installed `fioctl` and ran the configure credential helper command. It succeeded, then ran `git clone https://source.foundries.io/factories/<factory>/containers.git` and it failed. 

Then I started working the `.gitconfig` file to make the helper into an absolute path rather than just a name that might or might not resolve. With the full absolute path it worked consistently.
